sources/lib/structures/errors.ml

```diff
@@ -185,7 +185,7 @@ let report fmt = function
 
 let error e l = raise (Error(e,l))
 let warning e l =
-  if Options.output_smtlib () then Format.fprintf err_formatter "; ";
+  if Options.output_format () = Options.Smtlib2 then Format.fprintf err_formatter "; ";
```

Comment:
As mentionned by @OCamlPro-mattiasdrp this shouldn't be allowed, better to match on the input format, and possibly add a comment specifying why we need to add this in the case of smtlib2 I guess it's for the location to be in a commented line ?

Reply:
Done